guide/contribute/contribute.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: guide
+title: Contribute
+description: Additional material for both readers and writers of the guide.
+nav_order: 9
+has_children: true
+permalink: /guide/contribute/
+image: /assets/images/guide/contribute/resources-preview.jpg
+---
+
+# Contribute to the Guide
+
+People around the internet build and shape the Bitcoin Design Guide every day. You can be the part of our open-community too!
+
+This page shows you how to get involved and start contributing.
+
+---
+
+**[Content guidelines]({{ '/guide/contribute/content-guidelines/' | relative_url }})**
+
+Tips on how to write for the guide, with the goal of achieving a consistent tone across all pages.
+
+---
+
+**[Formatting]({{ '/guide/contribute/formatting/' | relative_url }})**
+
+Layout and formatting options available for content pages, from headers and blockquotes to images and embeds.
+
```

Comment:
```suggestion

---

```

Add a separator so the note about wider contribution does not visually merge with the description of the Formatting page.